docs/userguide/boundaryanalysis/index.md

```diff
@@ -80,14 +80,22 @@ different languages.
 Here's an example of a sentence, showing the boundary locations that will be
 identified by a word break iterator:
 
+<!--
```

Comment:
Add something to show that the example is missing. That shows in the rendered result. The example  was originally an image file; it seems to have gone missing in the old user guild as well.

Reply:
done.